docs/resources/sql_table.md

```diff
@@ -0,0 +1,91 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_sql_table (Resource)
+
+Within a metastore, Unity Catalog provides a 3-level namespace for organizing data: Catalogs, databases (also called schemas), and tables / views.
+
+A `databricks_sql_table` is contained within [databricks_schema](schema.md).
+
+## Example Usage
+
+```hcl
+resource "databricks_catalog" "sandbox" {
+  metastore_id = databricks_metastore.this.id
+  name         = "sandbox"
+  comment      = "this catalog is managed by terraform"
+  properties = {
+    purpose = "testing"
+  }
+}
+
+resource "databricks_schema" "things" {
+  catalog_name = databricks_catalog.sandbox.id
+  name         = "things"
+  comment      = "this database is managed by terraform"
+  properties = {
+    kind = "various"
+  }
+}
+
+resource "databricks_sql_table" "thing" {
+  provider           = databricks.workspace
+  name               = "quickstart_table"
+  catalog_name       = databricks_catalog.sandbox.id
+  schema_name        = databricks_schema.things.id
+  table_type         = "MANAGED"
+  data_source_format = "DELTA"
+  storage_location   = ""
+  column {
+    name      = "id"
+    position  = 0
+    type_name = "INT"
+    type_text = "int"
+    type_json = "{\"name\":\"id\",\"type\":\"integer\",\"nullable\":true,\"metadata\":{}}"
```

Comment:
to we want to keep it or do we want to suppress diff it?..

Reply:
@adamcain-db can we be fine with just this `type_json` from spark runtime?

Reply:
I settled on `type_text` here instead of the other fields, let me know if i need to support more.